subsys/bluetooth/controller/ll_sw/nordic/lll/lll_sync.c

```diff
@@ -635,7 +635,13 @@ static void abort_cb(struct lll_prepare_param *prepare_param, void *param)
 	lll = prepare_param->param;
 	lll->skip_prepare += (lll->lazy_prepare + 1U);
 
-	/* Reset Sync context association with any Aux context as the chain reception is aborted. */
+	/* Reset Sync context association with any Aux context as the chain reception is aborted.
+	 * There is no race with ULL execution context assigning lll_aux as this is prepare being
+	 * aborted and no assignment would be done yet before the superior PDU is received.
+	 * TODO: This code here is redundant as isr_rx_done_cleanup() should ensure it is reset
+	 *       on every periodic sync chain done. Remove it if there is no regression running
+	 *       conformance tests.
+	 */
```

Comment:
What is "the code here" referring to? The next line? More? 

If it's redundant, we should be able to remove it already. Claiming that it's redundant and still keeping it seems like an oxymoron 
Replace with an assert?